docs/reference/builder.md

```diff
@@ -1296,11 +1296,16 @@ into the newly created volume.
 
 ## USER
 
-    USER daemon
+    USER daemon[:wheel]
+
+The `USER` instruction sets the user name (or UID) and optionally the user
+group (or GID) to use when running the image and for any `RUN`, `CMD` and
+`ENTRYPOINT` instructions that follow it in the `Dockerfile`.
+
+> **Warning**:
+> When the user group is omited then it's possible that the image (or the next
+> instructions) will be running with the `root` group.
```

Comment:
> if only uid is provided, gid is reset to the default (0 - root)

It turns out that it happens not always but only in the case when user with provided uid doesn't have a primary group (4th field in the `/etc/passwd`/ file).

I've updated message. PTAL.